drivers/serial/usart_gd32.c

```diff
@@ -93,25 +148,201 @@ static int usart_gd32_err_check(const struct device *dev)
 	return errors;
 }
 
+#ifdef CONFIG_UART_INTERRUPT_DRIVEN
+/** Interrupt driven FIFO fill function */
+int usart_gd32_fifo_fill(const struct device *dev, const uint8_t *tx_data,
+			int len)
```

Comment:
`int len` should be vertically align with `const struct...`

Reply:
API declaration copy from drivers/uart.h.

Reply:
Done!

Reply:
Done!